pol-core/bscript/compiler/astbuilder/FunctionResolver.cpp

```diff
@@ -549,15 +550,15 @@ bool FunctionResolver::build_if_available( std::vector<AvailableParseTree>& to_b
 
       if ( handled_by_scope( cd->name ) )
       {
-        itr = to_link.end();
+        to_check_itr = to_check.end();
```

Comment:
Not needed

Reply:
Ah yup this was leftover from previous implementation